students/Gorthi_kavita/10module/10assignement.py

```diff
@@ -0,0 +1,173 @@
+class donar():
+    list =[]   #this is donar list adding  to the list
+    allist =[]   #to keep track how many times the donar name in the list
+    uniquename = []  # unique donar names
+    listsum = []     # donar names and total amount and number of times donated
+    listadd =[]
+    finalsum = []
+
+    def __data(self,name,amount):
+        self.donarname = name
+        self.donaramount = amount
+        c = (name,amount,1)
+        d = (name)
+        self.list.append(c)
+        self.allist.append(d)
+
+    def donarname(self):
+        self.__data("mike",8)
+        self.__data("ketty",75)
+        self.__data("john hoop",45)
+        self.__data("ravi",10)
+        self.__data("mike",6)
+        self.__data("joy",55)
+        self.__data("david",12)
+        self.__data("david",4)
+
+
+    def  countname(self):
+        import collections
```

Comment:
Best to import at the top of your function if you aren't worried about a circular import issue.  This way another developer knows right away all your dependencies.